### DISCUSSION TITLE
How can I change this hightlight?

### DISCUSSION THREAD

**Discussion by wangrongwei** [neoclide/coc.nvim#5156] [Q&A] [answered]
Hi folks

This hint always confused my in coding, how can I change it's style to make difference with general comment?
![image](https://github.com/user-attachments/assets/2965b438-71e1-4ad2-9076-956b461bc296)
(just an example)

Thanks for your help!

**Comment by chemzqm** [comment, marked as answer]
See `:h coc-highlights-diagnostics` highlight groups ends with `VirtualText`, `:h coc-highlights` for overwrite highlight groups.
